src/app/Library/CrudPanel/Traits/Create.php

```diff
@@ -66,11 +66,18 @@ public function getRelationFields()
                 array_push($relationFields, $field);
             }
 
-            if (isset($field['subfields']) &&
+            // if a field has an array name AND subfields
+            // then take those fields into account (check if they have relationships);
+            // this is done in particular for the checklist_dependency field,
+            // but other fields could use it too, in the future;
+            if (is_array($field['name']) &&
```

Comment:
Added another constraint here - only take `subfields` into consideration IF the field name is an array.